### ISSUE TITLE
citation.js can't initiate

### ISSUE THREAD

**Issue by naivenaive** [citation-js/citation-js#91]
I am going to use the citation.js to write an extension. First, I tried to write a sample html and use the demo displayed in [https://citation.js.org/](https://citation.js.org/). Here is the code: 

```
<!DOCTYPE html>
<html>
<head>
    <script src="https://cdn.jsdelivr.net/npm/citation-js" type="text/javascript"></script>  
</head>
<body>
    <h1>TEST</h1>
     <script>
    	const Cite = require('citation-js')
    	let example = new Cite('Q21972834')
        let output = example.format('bibliography', {
		  format: 'html',
		  template: 'apa',
		  lang: 'en-US'
		})
	console.log(output)
    </script>
</body>
</html>
```
In the console, I get the following error:

```
citation-js:1 Uncaught TypeError: Failed to construct 'Response': Please use the 'new' operator, this DOM object constructor cannot be called as a function.
    at new w (citation-js:1)
    at b (citation-js:1)
    at i (citation-js:1)
    at Array.map (<anonymous>)
    at Object.a (citation-js:1)
    at m (citation-js:1)
    at y (citation-js:1)
    at p.s [as add] (citation-js:1)
    at p [as set] (citation-js:1)
    at new p (citation-js:1)
```
Please help

**Comment by larsgw** [comment]
This seems to be caused by a tricky issue in Babel, which I can't fix right away. https://github.com/babel/babel/issues/10063

**Comment by larsgw** [comment]
This was fixed in `0.5.0-alpha.6`.
